src/test/ui/parser/recover-where-clause-before-tuple-struct-body.stderr

```diff
@@ -0,0 +1,51 @@
+error: where clauses are not allowed before tuple struct bodies
+  --> $DIR/recover-where-clause-before-tuple-struct-body.rs:4:1
+   |
+LL | / where
+LL | |     (): Sized,
+   | |______________^
+LL |   (usize);
+   |   ------- the tuple struct body
+   |
+help: consider moving the where clause after the body
+   |
+LL - where
+LL + (usize) where (): Sized;
+   |
```

Comment:
Wouldn't it look better to suggest moving the `(Type)` right next to the struct ident?

Reply:
We should also have a secondary span pointing at the ident in the main diagnostic to provide enough context in the diagnostic to understand at a glance which type this is talking about.

Reply:
Good idea! I'm going to try that! :)